GoMeet/src/main/webapp/test/spec/spec-interactive-map.js

```diff
@@ -225,10 +287,17 @@ describe ('Display Popular Location', function() {
     expect(childNodes[0].textContent).toBe('There are no locations to display.');
   });
 
-  it ('Should handle error is MeetingLocationDAO throws an error', async function() {
+  it ('Should handle error is PermMeetingLocationDAO throws an error', async function() {
+    // Spy on Dao.  
+    const mockedLocationDao = new PermMeetingLocationDAO();
```

Comment:
nit - same as above

Reply:
I think this should be lowerCamelCase. "Constants in function scopes are still named in lowerCamelCase."